app/common/templates/service-policies.j2

```diff
@@ -0,0 +1,6 @@
+path "{{secrets_root}}/{{owner}}/{{repo}}/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+
+path "identity/oidc/key/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+path "identity/oidc/role/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+path "auth/{{mount_point}}/role/*" { capabilities = ["create", "read", "update", "delete", "list"]}
+path "identity/oidc/token/*" { capabilities = ["create", "read", "update", "delete", "list"]}
```

Comment:
`/*` -- dangerous and powerful thing at the same time. Let's not forget to change it later.